CHANGELOG.md

```diff
@@ -62,10 +71,14 @@
 
 # 0.3.10 - 2021-01-13
 
+NOTE: This release has been yanked. See #2310 for details.
+
 * Fixed type-inference in `sink::unfold` by specifying more of its types (breaking change -- see #2311)
 
 # 0.3.9 - 2021-01-08
 
+NOTE: This release has been yanked. See #2310 for details.
```

Comment:
To be honest, I don't remember the exact reason why 0.3.9 and older releases were yanked, but #2310 is a proper reason to yank 0.3.10 and older releases, so I think this is okay.